doc/travis-ci.md

```diff
@@ -5,16 +5,27 @@ in a continuous manner, whenever new commits are pushed or on code from pull req
 
 We currently use Travis CI and thus prefer it for consistency, but other CIs are ok given reasonable consideration.
 
-Go to https://travis-ci.com/profile/tobiipro
+A very first step to setup an integration with Travis CI is to visit https://github.com/organizations/tobiipro/settings/installations,
+press `Configure` next to `Travis CI` and make sure that Travis
+has access to your repo in the `Repository access` section dropdown.
+
+Then go to https://travis-ci.com/profile/tobiipro
```

Comment:
What i noticed these days is that this is automatically done with the new integration described in the step above

Reply:
Still need to open the settings, will rephrase.